docs/serving/configuration/config-defaults.md

```diff
@@ -30,319 +32,497 @@ data:
+The `revision-ephemeral-storage-request` value determines the ephemeral storage
+allocation assigned to revisions by default. If this value is omitted, the system default is used. This key is not enabled by default for Knative.
 {% endraw %}
 
-**Key**: `revision-ephemeral-storage-request`
-
-**Default**: `"500M"` (500 megabytes of storage)
+* **Global key:** `revision-ephemeral-storage-request`
+* **Per-revision annotation key:** `serving.knative.dev/revision-ephemeral-storage-request`
+* **Possible values:** integer
+* **Default:** `"500M"` (500 megabytes of storage)
 
 **Example:**
-```yaml
-apiVersion:  v1
-kind:  ConfigMap
-metadata:
-  name:  config-defaults
-  namespace:  knative-serving
-data:
-  revision-ephemeral-storage-request: "500M" # 500 megabytes of storage
-```
 
-### Revision Cpu Limit
+=== "Global (ConfigMap)"
+    ```yaml
+    apiVersion:  v1
+    kind:  ConfigMap
+    metadata:
+      name:  config-defaults
+      namespace:  knative-serving
+    data:
+      revision-ephemeral-storage-request: "500M"
+    ```
+
+=== "Per Revision"
+    ```yaml
+    apiVersion: serving.knative.dev/v1
+    kind: Service
+    metadata:
+      name: helloworld-go
+      namespace: default
+    spec:
+      template:
+        metadata:
+          annotations:
+            serving.knative.dev/revision-ephemeral-storage-request: "500M"
+        spec:
+          containers:
+            - image: gcr.io/knative-samples/helloworld-go
+    ```
+
+### Revision CPU limit
 {% raw %}
-revision-cpu-limit contains the cpu allocation to limit
-revisions to by default.  If omitted, no value is specified
-and the system default is used.
-Below is an example of setting revision-cpu-limit.
-By default, it is not set by Knative.
+The `revision-cpu-limit` value determines the default CPU allocation limit for revisions. If this value is omitted, the system default is used. This key is not enabled by default for Knative.
 {% endraw %}
 
-**Key**: `revision-cpu-limit`
-
-**Default**: `"1000m"` (1 CPU (aka 1000 milli-CPU))
+* **Global key:** `revision-cpu-limit`
+* **Per-revision annotation key:** `serving.knative.dev/revision-cpu-limit`
+* **Possible values:** integer
+* **Default:** `"1000m"` (1 CPU, or 1000 milli-CPU)
 
 **Example:**
-```yaml
-apiVersion:  v1
-kind:  ConfigMap
-metadata:
-  name:  config-defaults
-  namespace:  knative-serving
-data:
-  revision-cpu-limit: "1000m" # 1 CPU (aka 1000 milli-CPU)
-```
 
-### Revision Memory Limit
+=== "Global (ConfigMap)"
+    ```yaml
+    apiVersion:  v1
+    kind:  ConfigMap
+    metadata:
+      name:  config-defaults
+      namespace:  knative-serving
+    data:
+      revision-cpu-limit: "1000m"
+    ```
+
+=== "Per Revision"
+    ```yaml
+    apiVersion: serving.knative.dev/v1
+    kind: Service
+    metadata:
+      name: helloworld-go
+      namespace: default
+    spec:
+      template:
+        metadata:
+          annotations:
+            serving.knative.dev/revision-cpu-limit: "1000m"
```

Comment:
Likewise this annotation doesn't exist - the config map property configures the  default value for `Spec.Containers[*].Limits.CPU` of a `Revision`

Reply:
see previous comments

Reply:
similar to above